en/hardware/porting_guide_config.md

```diff
@@ -30,6 +30,10 @@ if DRIVERS_UAVCAN_V1
 endif #DRIVERS_UAVCAN_V1
 ```
 
+:::note
+Builds will silently ignore any missing or miss-spelled modules in the  `*.px4board` configuration file.
```

Comment:
@hamishwillee Shouldn't we add the *if edited manually part here? As it wouldn't happening using kconfig gui utilities?

I spent my fair share of time just troubleshooting this because of this, since I didn't expect the menuconfig name to be the one included in the .px4board file.

(I thought CONFIG_*) ones were parsed automatically to access the modules in certain directories, and didn't know I had to manually edit the Kconfig to match the directory naming convention exactly 🙊

Reply:
The note highlights the result of both missing and miss-spelled modules.
So if this is not sufficient then I think that we'd be better off with a statement like "Don't edit the files manually".